osbrain/agent.py

```diff
@@ -912,7 +922,7 @@ def _handle_async_requests(self, data):
         else:
             handler(self, response)
 
-    def _subscribe(self, alias: str, handlers: Dict[Union[bytes, str], Any]):
+    def subscribe(self, alias: str, handlers: Dict[Union[bytes, str], Any]):
         """
         Subscribe the agent to another agent.
 
```

Comment:
It would be great to complete de docstring here as well now that the method is "user accessible".